docs/learning/CommunityResources.md

```diff
@@ -14,7 +14,7 @@ See the CONTRIBUTING.md file in the root of the imodeljs repository for addition
 
 ### Questions and Advice
 
-Programming questions and requests for programming advice are best asked on [Stack Overflow](https://stackoverflow.com/tags/imodeljs).
+Programming questions and requests for programming advice are best asked on [Stack Overflow](https://stackoverflow.com/tags/bentley-itwin).

```

Comment:
@kabentley is it time to remove this "best asked on stack overflow" advice? I never look there, virtually nobody asks there, and GitHub discussions are for more conducive to providing programming advice.

Reply:
[Case in point](https://stackoverflow.com/questions/67591825/how-can-i-read-bentley-pod-file). Do we really need stackoverflow randos deciding what constitutes a worthy question?

Reply:
yes, try as i might i have never been able to get stackoverflow notifications to work. Let's just remove references to it.

Reply:
@Josh-Schifter I just removed the mention of StackOverflow for iTwin.js based questions and updated all links to Discussions. This follows the similar pattern to how we mention support on https://developer.bentley.com/support/.

I followed up internally on the question around notifications for SO if we continue to use it for iTwin Platform more specifically.

Reply:
@pmconne mind taking a look at the changes made?